src/benchmarklib/tpch/tpch_db_generator.cpp

```diff
@@ -62,12 +62,16 @@ class TableBuilder {
   TableBuilder(size_t chunk_size, const boost::hana::tuple<ColumnTypes...>& column_types, const boost::hana::tuple<Strings...>& column_names) {
     _table = std::make_shared<opossum::Table>(chunk_size);
 
-    boost::hana::zip_with(
+    const auto column_names_and_data_types = boost::hana::zip_with(
       [&](auto column_type, auto column_name) {
-        _table->add_column_definition(column_name, opossum::data_type_from_type<decltype(column_type)>());
-        return 0; // Can't return void, because that makes boost::hana unhappy
+        return boost::hana::make_tuple(column_name, opossum::data_type_from_type<decltype(column_type)>());
       },
       column_types, column_names);
+
+    boost::hana::fold_left(column_names_and_data_types, _table, [](auto table, auto column_name_and_type) {
+      table->add_column_definition(column_name_and_type[boost::hana::llong_c<0>], column_name_and_type[boost::hana::llong_c<1>]);
```

Comment:
I think you can even write `column_name_and_type[0_c]` http://www.boost.org/doc/libs/1_61_0/libs/hana/doc/html/structboost_1_1hana_1_1integral__constant.html#a1c078fd00db48686ca9e0aa17f47344e

Reply:
That would require a `using namespace boost::hana::literals` and lint won't allow that

Reply:
Okay don’t tell this the others, but there is trick. Try adding `// NOLINT` behind the line, but don’t tell anyone that I told you to do so 🤐

Reply:
I'm not sure we should do this, either be strict with style requirements or don't have them, I guess? @mrks?

Reply:
In very rare cases, nolint can be fine. Here, I don't think that the change is important enough. Also, I am not sure how many people know about UDLs already. Wouldn't use them if I don't have to.